src/commands/anime/anime.js

```diff
@@ -51,31 +54,120 @@ function getAnimeInfos(anime) {
+	return animesFields;
+}
+
+function showListOfAnime(msg,animesList){
+	msg.channel.send({
+		embed: {
+			title: 'List of animes corresponding to the research',
+			fields: getAnimesListInfos(animesList),
+		},
+	}).then(async postedMessage => {
+		try {
+			const filter = (reaction,user) => {
+				const emojiAccepted = [];
+				for(var i = 0; i < animesList.length; i++){
+					emojiAccepted.push(numberInEmojiUnicodeUntilTen(i+1));
+				}
+				return emojiAccepted.includes(reaction.emoji.name) && user.id === msg.author.id;
+			};
+			postedMessage.awaitReactions(filter, {max: 1, time: 60000, errors:['time'] })
+				.then(collected => {
+					const reaction = collected.first();
+		
+					switch(reaction.emoji.name){
+						case numberInEmojiUnicodeUntilTen(1) :
+							sendAnimeInfos(msg,animesList[0]);
+							break;
+						case numberInEmojiUnicodeUntilTen(2) :
+							sendAnimeInfos(msg,animesList[1]);
+							break;
+						case numberInEmojiUnicodeUntilTen(3) :
+							sendAnimeInfos(msg,animesList[2]);
+							break;
+						case numberInEmojiUnicodeUntilTen(4) :
+							sendAnimeInfos(msg,animesList[3]);
+							break;
+						case numberInEmojiUnicodeUntilTen(5) :
+							sendAnimeInfos(msg,animesList[4]);
+							break;
+						case numberInEmojiUnicodeUntilTen(6) :
+							sendAnimeInfos(msg,animesList[5]);
+							break;
+						case numberInEmojiUnicodeUntilTen(7) :
+							sendAnimeInfos(msg,animesList[6]);
+							break;
+						case numberInEmojiUnicodeUntilTen(8) :
+							sendAnimeInfos(msg,animesList[7]);
+							break;
+						case numberInEmojiUnicodeUntilTen(9) :
+							sendAnimeInfos(msg,animesList[8]);
+							break;
+						case numberInEmojiUnicodeUntilTen(10) :
+						sendAnimeInfos(msg,animesList[9]);
+						break;
+						default : msg.reply('unknown reaction');
+					}
+				}).catch(collected => {
+					console.error(collected);
+					msg.reply('you didn\'t react with an allowed reaction in time. Please retry the command and react in time by following instructions.')
+				});
+		} catch (error) {
+			console.error('One of the emojis failed to react.');
+		}
+	});
+	
+}
+
 exports.exec = (bot, msg, args) => {
 	const Kitsu = new KitsuAPI();
 	Kitsu.findByAnimeName(args.join(' '))
 		.then(response => {
 			if(response.status == 200) {
 				if(response.data.data.length > 0) {
-					const anime = response.data.data[0];
-					msg.channel.send({
-						embed: {
-							title: `:flag_gb: ${anime.attributes.titles.en_jp}\n:flag_jp: ${anime.attributes.titles.ja_jp}`,
-							image: {
-								url: `${anime.attributes.coverImage != null ? anime.attributes.coverImage.original : ''}`,
-							},
-							thumbnail: {
-								url: `${anime.attributes.posterImage != null ? anime.attributes.posterImage.original : ''}`,
-							},
-							fields: getAnimeInfos(anime),
-							description: `**Synopsis : **\n\n${anime.attributes.synopsis.split('[')[0].split('(Source')[0]}`,
-						},
-					});
+					showListOfAnime(msg,response.data.data);
```

Comment:
In the case when there is only one anime in the list, can you display it directly ?
Something like : 
```
const animes = response.data.data;
if (animes.length === 1) {
    sendAnimeInfos(animes[0]);
}
else if (animes.length > 1) {
    showListOfAnime(msg, animes);
}
else {
    // ...
}
```